jsonschema/validators.py

```diff
@@ -438,7 +438,8 @@ def extend(validator, validators=(), version=None, type_checker=None):
         u"anyOf": _validators.anyOf,
         u"const": _validators.const,
         u"contains": _validators.contains,
-        u"dependencies": _validators.dependencies,
```

Comment:
AIUI, 'dependencies' is still a valid keyword even if deprecated. It's still in the meta-schema.

Reply:
I'm not sure, but according to the [migration guide](https://json-schema.org/draft/2019-09/release-notes.html) to `2019-19` this has been split to `dependentSchemas` and `dependentRequired`. The test cases have also been removed from [JSON-Schema-Test-Suite](https://github.com/json-schema-org/JSON-Schema-Test-Suite) for the newer drafts.

The meta-schema reserves the keyword for backwards compatibility, so I assume that the implementation decides if this should be supported or not. @Julian any suggestion?

Reply:
It's meant to still be supported via an optional compatibility mode, not really by default IIRC.

So yeah I'd remove them for now. We can choose to add the compatibility mode later if we want, but I also think I agree (with the spec) that it's trivial to use the new properties so it doesn't seem very high value even as an opt-in.

Reply:
In this case, should we move the `dependencies` validation function to `_legacy_validators.py`

Reply:
Yes. Basically any validator that doesn't apply to the latest draft I move to the legacy file.

Reply:
The function has been moved to `_legacy_validators.py`